styles/package.json

```diff
@@ -1,14 +1,14 @@
 {
   "devDependencies": {
     "bootstrap": "^4.6.1",
-    "node-sass": "^7.0.0"
+    "sass": "^1.69.5"
   },
   "scripts": {
     "build-css": "node-sass --include-path scss xrpl.scss ../assets/css/devportal2022-v25.css --output-style compressed",
     "build-css-out": "node-sass --include-path scss xrpl.scss ../out/assets/css/devportal2022-v25.css --output-style compressed --source-map true",
     "build-css-watch-out": "node-sass --recursive --watch  --include-path scss xrpl.scss ../out/assets/css/devportal2022-v25.css --output-style compressed --source-map true"
   },
   "dependencies": {
-    "sass": "^1.26.10"
+    "node-sass": "^9.0.0"
```

Comment:
Why were these swapped?

Reply:
`node-sass` is deprecated.  `sass` should be used instead.

Reply:
reverted the changes